client/layout/global-sidebar/hooks/use-global-sidebar.js

```diff
@@ -2,12 +2,14 @@ import { isEnabled } from '@automattic/calypso-config';
 
 export const useGlobalSidebar = ( siteId, sectionGroup ) => {
 	let shouldShowGlobalSidebar = false;
+	let shouldShowGlobalSiteViewSidebar = false;
```

Comment:
[super nit] I think these should be either

- shouldShowGlobalViewSidebar
- shouldShowGlobalSiteViewSidebar

or

- shouldShowGlobalSidebar
- shouldShowGlobalSiteSidebar

what do you think?

Reply:
> shouldShowGlobalSidebar
shouldShowGlobalSiteSidebar

I'll choose this one, as I thought "View" is just redundant. Thanks for your comment!